### PR TITLE
script to divest from sushi and invest cvx

### PR DESCRIPTION
**Pull request by jalbrekt85**
addresses #255, provides functions to transfer, unwrap and sell sushi for cvx. sushi sold according to strat outlined [here](https://github.com/Badger-Finance/badger-multisig/issues/255#issuecomment-1068949864)

### CODE REVIEW

**Comment by jorijnsmit** [comment]
@jalbrekt85 did you find my pr https://github.com/jalbrekt85/badger-multisig/pull/1 ?

**Review by jorijnsmit — DISMISSED** [review]
reviewed, but wrote the cowswap part myself. please have a good look at it.

File: scripts/issue/255/divest_xsushi.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+def post_orders_to_cowswap():
+    TROPS.init_cow()
+    TROPS.cow.allow_relayer(SUSHI, SUSHI.balanceOf(TROPS))
+    half = int(SUSHI.balanceOf(TROPS) / 2)
+    a_tenth = int(SUSHI.balanceOf(TROPS) / 10)
+    rest = SUSHI.balanceOf(TROPS) - half - 4 * a_tenth
+    assert half + 4 * a_tenth + rest == SUSHI.balanceOf(TROPS)
+    TROPS.cow.market_sell(SUSHI, CVX, half, deadline=60*60*24)
+    TROPS.cow.market_sell(SUSHI, CVX, a_tenth, deadline=60*60*24*10, coef=.99)
```
Comment (petrovska-petro) [inline scripts/issue/255/divest_xsushi.py:40]: here param `coef` should be the other way around if we want to get better prices, here we are accepting on each slightly worst pricing laddering down

from coef=1.01 to coef=1.05 i believe

**Comment by jorijnsmit** [comment]
@jalbrekt85 note that we are trying to sell for above the market price, so at 101% of the quote received from the api. `coef` should therefore be set to `1.01`, `1.02`, etc. my old code was mistakenly selling for 1%, 2%, 3% etc. under the market price instead.

**Review by jorijnsmit — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]
Great, looking forward to see how it plays out this 1st ladder order 👀

**Comment by jorijnsmit** [comment]
https://gnosis-safe.io/app/eth:0x042B32Ac6b453485e357938bdC38e0340d4b9276/transactions/0xe54b6a0ab82c286ac233e430d71015d6581f1d451d25386eec75f2d9884122b3
```
Events In This Transaction
--------------------------
├── SushiBar (0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272)
│   └── Transfer
│       ├── from: 0x042B32Ac6b453485e357938bdC38e0340d4b9276
│       ├── to: 0x0000000000000000000000000000000000000000
│       └── value: 127937510720383332404871
│   
└── Gnosis Safe (0x042B32Ac6b453485e357938bdC38e0340d4b9276)
    └── ExecutionSuccess
        ├── txHash: 0xe54b6a0ab82c286ac233e430d71015d6581f1d451d25386eec75f2d9884122b3
        └── payment: 0
```
```
{
│   'ethereum_client': <gnosis.eth.ethereum_client.EthereumClient object at 0x11e233a90>,
│   'safe_address': '0x042B32Ac6b453485e357938bdC38e0340d4b9276',
│   'to': '0x40A2aCCbd92BCA938b02010E17A5b8929b49130D',
│   'value': 0,
│   'data': HexBytes('0x8d80ff0a00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000079008798249c2e607446efb7ad49ec89dd1865ff42720000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002467dfd4c9000000000000000000000000000000000000000000001b1781a05e34b2784e8700000000000000'),
│   'operation': 1,
│   'safe_tx_gas': 0,
│   'base_gas': 0,
│   'gas_price': 0,
│   'gas_token': '0x0000000000000000000000000000000000000000',
│   'refund_receiver': '0x0000000000000000000000000000000000000000',
│   'signatures': b'',
│   '_safe_nonce': 57,
│   '_safe_version': '1.3.0',
│   '_chain_id': None
}
```
```
snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
                    balance_before               balance_after                balance_delta
symbol                                                                                     
xSUSHI  127,937.510720383332404871        0.000000000000000000  -127,937.510720383332404871
SUSHI         0.000000000000000000  162,502.065035676392446071   162,502.065035676392446071 
```

**Comment by jorijnsmit** [comment]
```
snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
                   balance_before         balance_after               balance_delta
symbol                                                                             
CVX     24,729.253494666585854953  0.000000000000000000  -24,729.253494666585854953 

snapshot result for 0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e:
              balance_before              balance_after              balance_delta
symbol                                                                            
bveCVX  0.000000000000000000  24,729.253494666585854953  24,729.253494666585854953 
```
```
Events In This Transaction
--------------------------
├── Badger Vested Escrow Convex Token (0xfd05D3C7fe2924020620A8bE4961bBaA747e6305)
│   └── Transfer
│       ├── from: 0x0000000000000000000000000000000000000000
│       ├── to: 0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e
│       └── value: 24729253494666585854953
│   
└── Gnosis Safe (0x042B32Ac6b453485e357938bdC38e0340d4b9276)
    └── ExecutionSuccess
        ├── txHash: 0x418d6ba72f16576aeb9c7ac4ac624922fd8be459836655295c69ce62f69e370c
        └── payment: 0
```
```
{
│   'ethereum_client': <gnosis.eth.ethereum_client.EthereumClient object at 0x120b7ca90>,
│   'safe_address': '0x042B32Ac6b453485e357938bdC38e0340d4b9276',
│   'to': '0x40A2aCCbd92BCA938b02010E17A5b8929b49130D',
│   'value': 0,
│   'data': HexBytes('0x8d80ff0a00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000132004e3fbd56cd56c3e72c1403e103b45db9da5b9d2b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044095ea7b3000000000000000000000000fd05d3c7fe2924020620a8be4961bbaa747e630500000000000000000000000000000000000000000000053c93552832ed1253e900fd05d3c7fe2924020620a8be4961bbaa747e6305000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000442f4f21e2000000000000000000000000d0a7a8b98957b9cd3cfb9c0425abe44551158e9e00000000000000000000000000000000000000000000053c93552832ed1253e90000000000000000000000000000'),
│   'operation': 1,
│   'safe_tx_gas': 0,
│   'base_gas': 0,
│   'gas_price': 0,
│   'gas_token': '0x0000000000000000000000000000000000000000',
│   'refund_receiver': '0x0000000000000000000000000000000000000000',
│   'signatures': b'',
│   '_safe_nonce': 68,
│   '_safe_version': '1.3.0',
│   '_chain_id': None
}
```

**Pull request merged**